netlify.toml

```diff
@@ -1,2 +1,2 @@
 [build]
-  command = "NEXT_PUBLIC_BASE_URL=$URL yarn build"
+  command = "NEXT_PUBLIC_BASE_URL=\"${URL:-$URL}\" yarn build"
```

Comment:
Should the first `URL` be `NEXT_PUBLIC_BASE_URL `

Reply:
no

Reply:
maybe

Reply:
will fix, thanks